README.md

```diff
@@ -65,6 +72,7 @@ PRs are expected to pass the following across the organization
 - Unit test coverage
 - CI
 - PR Line size check
+- PR semantic format checker
```

Comment:
I would also expect we have linting on the commits here with a husky hook or something of that nature.  Commit format doesn't take long to get into the muscle memory, but anything we can automate away from the discussion is helpful to keep things focused on the right things.

Reply:
Sounds good. Semantically, would it be more accurate to call this "Lint commit hook"?